client/store/index.js

```diff
@@ -1,15 +1,16 @@
-import {createStore, combineReducers, applyMiddleware} from 'redux'
-import createLogger from 'redux-logger'
-import thunkMiddleware from 'redux-thunk'
-import {composeWithDevTools} from 'redux-devtools-extension'
-import user from './user'
+import {createStore, combineReducers, applyMiddleware} from 'redux';
+import createLogger from 'redux-logger';
+import thunkMiddleware from 'redux-thunk';
+import {composeWithDevTools} from 'redux-devtools-extension';
+import user from './user';
+import productReducer from './products';
 
-const reducer = combineReducers({user})
+const reducer = combineReducers({user, productReducer});
 const middleware = composeWithDevTools(applyMiddleware(
   thunkMiddleware,
   createLogger({collapsed: true})
-))
-const store = createStore(reducer, middleware)
+));
+const store = createStore(reducer, middleware);
 
-export default store
-export * from './user'
+export default store;
+export * from './user';
```

Comment:
Do you need to export * from './product' perhaps in order to get your thunks

Reply:
yes, if we want to be able to export all of our reducer stuff from this one file we should be consistent with that